quickannotator/api/v1/tile/models.py

```diff
@@ -0,0 +1,48 @@
+from marshmallow import fields, Schema
+from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+import quickannotator.db.models as db_models
+
+
+class TileRespSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = db_models.Tile
+        include_fk = True
```

Comment:
what is fk?

Reply:
foreign key?

Reply:
Yes, this is the pattern for configuring the SqlAlchemyAutoSchema class, and here I specify to include foreign keys in the output. The foreign keys are annotation_class_id and image_id, which are needed in combination with tile_id to access the idx_annotation_class_image_tile index on Tile.

Here's the documentation for include_fk
https://marshmallow-sqlalchemy.readthedocs.io/en/latest/api_reference.html#marshmallow_sqlalchemy.SQLAlchemyAutoSchemaOpts